extensions-contrib/sqlserver-metadata-storage/src/test/java/org/skife/jdbi/v2/CustomStatementRewriterTest.java

```diff
@@ -0,0 +1,130 @@
+package org.skife.jdbi.v2;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.skife.jdbi.v2.exceptions.UnableToCreateStatementException;
+import org.skife.jdbi.v2.tweak.RewrittenStatement;
+
+import io.druid.metadata.storage.sqlserver.SQLServerConnector.CustomStatementRewriter;
+import junit.framework.Assert;
+
+@SuppressWarnings( "nls" )
+public class CustomStatementRewriterTest
```

Comment:
I wanted to to instantiate a `StatementRewriter` instance and to do this I constructed a `ConcreteStatementContext` class, which unfortunately is final and has package level access to the constructor.

However, I played with the code a bit and noticed that the `ColonPrefixNamedParamStatementRewriter` (`StatementRewriter`) can survive with a null `ConcreteStatementContext` value.  Will update.